fee_calculator/apps/calculator/tests/base.py

```diff
@@ -147,3 +147,46 @@ def create_tests(cls):
         print('{0}: Testing {1} scenarios'.format(
             cls.__name__, len(tested_scenarios)
         ))
+
+
+class EvidenceProvisionFeeTestMixin():
+
+    def test_evidence_provision_fee_0(self):
+        data = {
+            'scheme': self.scheme_id,
+            'fee_type_code': 'EVID_PROV_FEE',
+            'scenario': 4,
+            'offence_class': 'A',
+            'level': 0
+        }
+        self.check_result(data, Decimal(0))
+
+    def test_evidence_provision_fee_1(self):
+        data = {
+            'scheme': self.scheme_id,
+            'fee_type_code': 'EVID_PROV_FEE',
+            'scenario': 4,
+            'offence_class': 'A',
+            'level': 1
+        }
+        self.check_result(data, Decimal(45))
+
+    def test_evidence_provision_fee_2(self):
+        data = {
+            'scheme': self.scheme_id,
+            'fee_type_code': 'EVID_PROV_FEE',
+            'scenario': 2,
+            'offence_class': 'B',
+            'level': 2
+        }
+        self.check_result(data, Decimal(90))
+
+    def test_evidence_provision_fee_3(self):
+        data = {
+            'scheme': self.scheme_id,
+            'fee_type_code': 'EVID_PROV_FEE',
+            'scenario': 1,
+            'offence_class': 'D',
+            'level': 3
```

Comment:
as discussed, i think an erroroneous level should return 0 in case we need to handle such things in an API consumer.